vertical-pod-autoscaler/pkg/utils/test/test_vpa.go

```diff
@@ -131,6 +133,12 @@ func (b *verticalPodAutoscalerBuilder) WithTarget(cpu, memory string) VerticalPo
 	return &c
 }
 
+func (b *verticalPodAutoscalerBuilder) WithResourceInTarget(resource core.ResourceName, value string) VerticalPodAutoscalerBuilder {
```

Comment:
Let's call it the same as the function in vertical-pod-autoscaler/pkg/utils/test/test_recommendation.go

Also please add a TODO, it looks like `verticalPodAutoscalerBuilder` (here) and `recommendationBuilder` (in test_recommendation.go) could use some deduplication

Reply:
done.